packages/node_modules/@webex/webex-core/test/integration/spec/services/services.js

```diff
@@ -535,6 +535,28 @@ describe('webex-core', () => {
             done();
           });
       });
+      it('updates the limited catalog when query param mode is provided', (done) => {
+        catalog.serviceGroups.preauth = [];
+
+        services.updateServices({
+          from: 'limited',
+          query: {mode: 'DEFAULT_BY_PROXIMITY'}
+        })
+          .then(() => {
+            assert.isAbove(catalog.serviceGroups.preauth.length, 0);
+            done();
+          });
+      });
+      it('does not update the limited catalog when nothing is provided', (done) => {
+        catalog.serviceGroups.preauth = [];
+
+        services.updateServices({
+        })
+          .then(() => {
+            done();
```

Comment:
You should call the `done` function after your assertions. You can also just return the promise above if you would rather do that.